sky/serve/serve_utils.py

```diff
@@ -731,10 +731,22 @@ def format_service_table(service_records: List[Dict[str, Any]],
             f'{replica_table}')
 
 
+def _sort_replica_records(
+        replica_records: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+
+    def sort_key(record: Dict[str, Any]) -> Tuple[int, int]:
```

Comment:
```suggestion
    def _sort_key(record: Dict[str, Any]) -> Tuple[int, int]:
```
nit: make it private